### PR TITLE
Feature/language selection

### PR DESCRIPTION
**Pull request by michaelWuensch**
## Description

I added all languages to the languages selection option which had >75% translated. Which are:

Bulgarian
Czech
English
French
German
Italian
Spanish

The language options themselves are not translatable. "System Language" is translatable.

## Motivation and Context

Giving people the ability to select their language even if they still have english as their system language.
Address #129 and #149 

## How Has This Been Tested?

On my S9

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [Contribution document](../docs/CONTRIBUTING.md).
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by Relaxo143** [comment]
@michaelWuensch Tested on Android Studio emulator, everything seems to be working as expected, no crashes. I think the language options themselves should be translatable, but English would do just fine as well. Maybe we can even do like "Deutsch (German)"

**Comment by michaelWuensch** [comment]
@Relaxo143 I can only make all language names translatable or none. The problem is, that this would be different for every language if I want to make just that one translatable. So if I wanted to have "German" as "Deutsch" I would also have to translate "Bulgarian" to "Bulgarisch", etc.
I actually would not have to translate all, but then crowdin would show the translation as unfinished.

**Comment by Relaxo143** [comment]
@michaelWuensch Maybe you could just hard-code them (outside of the strings files), or leave them in English alternatively

**Comment by michaelWuensch** [comment]
@Relaxo143 I think I prefer English. Someone that has set his phone language to English should'n have problems to find his own language in English. And if they haven't, there is no need to switch anyway, as the default is the system language.

File: docs/TRANSLATING.md [hunk 1]
```diff
@@ -12,5 +12,6 @@
 
 Your translation will only be visible in the app after the next update will be published on Google Play Store. There is no fixed schedule when updates are happening.
 So this might take a few days or a few weeks.
+If the language is translated to 100%, we will add it to the language selection list. If not, it is onyl accessible if you choose "System Language" and your smartphone is set to the corresponding language.
```
Comment (raphBTC) [inline docs/TRANSLATING.md]: `only` typo

**Pull request merged**